airflow/providers/elasticsearch/provider.yaml

```diff
@@ -66,7 +66,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.7.0
-  - apache-airflow-providers-common-sql>=1.14.1
+  - apache-airflow-providers-common-sql>=1.14.2
```

Comment:
We need the versions updated as the PR was not merged in time for the current release wave

Reply:
Indeed, and while doing that I just saw there is an issue as the code isn't up-to-date with the previous changes I did that are already merged into main, so I'm a bit confused.  When I compare untouched files there are differences with main but when I do a pull rebase no new commits or changes are comming in it says like up-to-date...  When I do git merge main also no changes applied

Reply:
Never mind I think PyCharm glitched or something